src/NixTree/Main.hs

```diff
@@ -64,6 +65,15 @@ optsParser =
                     )
               )
           )
+        <*> optional
+          ( Opts.strOption
+              ( Opts.long "file"
+                  <> Opts.metavar "FILE"
+                  <> Opts.helpDoc
+                    ( Just $ Opts.vsep ["File used by `nix path-info" ]
```

Comment:
I feel like this leaks implementation details a bit - the user might not know whether/how `nix-tree` uses `nix path-info`. Maybe we should ~steal~ use their explanation instead:

```
Interpret installables as attribute paths relative to the Nix expression stored in file.
```

Reply:
done